src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,8 @@ public Order completeOrder(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Data from DB were successfully fetched. "
+                + "Params: userId = {}, amount of purchases = {}", userId, products.size());
```

Comment:
```suggestion
                + "Params: userId = {}, products = {}", userId, products);
```
products will be more important information than amount of purchases